specs/use_value_attribute_to_locate_object_paths_in_etcd.adoc

```diff
@@ -0,0 +1,104 @@
+// vim: tw=79
+
+= Use 'value' attribute in objects to locate object path in etcd
+
+Tendrl API should use the 'value' attribute defined in the definitions yaml
+files to infer the path of the object in etcd. These paths are used to retrieve
+a single instance or a list of the defined object.
```

Comment:
"value" will point to a single instance of the object, and "list" will point to the list of instances of those object (see: https://github.com/Tendrl/specifications/pull/13)